draft-ietf-tls-tls13.md

```diff
@@ -1010,17 +1010,14 @@ of {{RFC5116}}. The key is either the client_write_key or the server_write_key.
 
 %%% Record Layer
        struct {
-           ContentType type;
```

Comment:
I thought the plan was just to do all the changes and see if anything important breaks that needs a kludge to deal with. I think you said this at one of the meetings, if I'm remembering from the video correctly.